docs/src/how-to/install/web-app-settings.md

```diff
@@ -44,3 +44,17 @@ webapp:
     # ...
     FEATURE_ENABLE_MEDIA_EMBEDS: "false"
 ```
+
+## Enable extra entropy
+
+The Wire desktop application uses system-dependent pseudo-random number generators (PRNG) as an internal entropy source when generating cryptographic keys. In certain cases it may be desired to enable externally generated entropy derived from mouse movement.
```

Comment:
You probably copied all this from somewhere?  Should the source be replaced with a link?

Also: I think it's inaccurate to describe the source of entropy as *P*RNG.  These usually take truly random system events and the PRNG only runs over the data provided from those to provide more bits when needed.   What about this:

```suggestion
The Wire desktop application uses system-dependent source of random bits as an internal entropy source when generating cryptographic keys. In certain cases it may be desired to enable externally generated entropy derived from mouse movement.
```